subprojects/docs/src/docs/userguide/authoring-builds/authoring_maintainable_build_scripts.adoc

```diff
@@ -241,3 +241,45 @@ resides in the user's home directory or be injected to the build using command l
 
 If you store sensitive credentials in user home's `gradle.properties`, consider encrypting them. At the moment Gradle does not provide a built-in mechanism for encrypting, storing and accessing passwords.
 A good solution for solving this problem is the link:https://github.com/etiennestuder/gradle-credentials-plugin[Gradle Credentials plugin].
+
+[[sec:dont_anticipate_configuration_creation]]
+== Don't anticipate configuration creation
+
+Gradle will create certain configurations using a "check if needed" strategy.
+That means it will only create those configurations if the **do not** already exist.
+
+You should **not** create these configurations yourself.
```

Comment:
I think this line should be moved below the two below it. When I read this line I think "which configurations are `these configurations`"?

Then, below, I think "ahh, `these configurations` are `default`, `archives`, etc"

Reply:
I didn't want to pretend like I could supply an exhaustive list here.  I'll move these examples first though, so that "these" points to something at least.